.gitignore

```diff
@@ -4,3 +4,4 @@ npm-debug.log
 *.swp
 *.swo
 *.pyc
+yarn.lock
```

Comment:
Please remove line this for now.
(Btw, if you are interested -- you could file another PR replacing the use of node with yarn)